resources/upgrade-notification.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+FROM_VERSION="${1}"
+TO_VERSION="${2}"
+
+# print major upgrade notification if TO_VERSION is equal or higher than 12.5-1 and FROM_VERSION is lower than 12.5-1
```

Comment:
This is a replica of my finding above, for the same reason.

For compatibility reasons a duplicated function does not hurt (you most probably cannot share the function because before the upgrade only this file is extracted and not any utility files.

- [ ]  done

Reply:
As I am not working with simple integers here, I wouldn't change the way the versions are compared. Therefore I see no improvement in extracting the function other than maybe getting rid of the comment.